libraries/abstractions/secure_sockets/freertos_plus_tcp/iot_secure_sockets.c

```diff
@@ -210,6 +210,17 @@ uint32_t SOCKETS_GetHostByName( const char * pcHostName )
 {
     return FreeRTOS_gethostbyname( pcHostName );
 }
+
+
+/*-----------------------------------------------------------*/
+
+int32_t SOCKETS_Bind( Socket_t xSocket,
+                      SocketsSockaddr_t *pxAddress,
+                      Socklen_t xAddressLength )
+{
+   return SOCKETS_SOCKET_ERROR;
```

Comment:
Please put the following to avoid ununsed variables warning:

```c
( void ) xSocket;
( void ) pxAddress;
( void ) xAddressLength;
```

Reply:
I have done this change, resolving conversation